pyhf/interpolate.py

```diff
@@ -1,7 +1,7 @@
 import logging
 log = logging.getLogger(__name__)
 
-from . import get_backend
+from . import get_backend, default_backend
```

Comment:
should we give this another name? we could at some point change the default to TF but we would still not want those tensors in __init__ in TF.. it's like a  "prep backend" / "dark backend" or sth

Reply:
our tests rely on resetting the backend. I suppose if we switch it to tensorflow, then we need to explicitly change it to use numpy... I can't imagine the default backend would ever change to TF though, especially since like mxnet depends on numpy -- and this is all set up based on how our backends get imported (always require numpy).

Reply:
ok let's stay with `default_backend` if we at some point dont' want to use our default for the prep stuff we can revisit later